ui/src/main/webapp/src/app/app.routing.ts

```diff
@@ -71,16 +71,15 @@ export const appRoutes: Routes = [
                 component: DefaultLayoutComponent,
                 children: [
                     {path: '', redirectTo: "/project-list", pathMatch: "full"},
+                    {path: 'about', component: AboutPageComponent},
                     {
                         path: "configuration",
-                        resolve: { configuration: ConfigurationResolve },
```

Comment:
The reformat is ok, it just would have been nice for it to be in its own commit. It's ok, though.

Also, we haven't been autoformatting these, so we have a lot of little inconsistencies. Maybe we can agree on a formatter and fix it all at once after the release.